docs/getting-started/add-sample-data.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[add-sample-data]]
+== Get up and running with sample data
+
+{kib} has three sample data sets that you can explore before loading your own data 
+source. Each set is prepackaged with a dashboard of visualizations and a 
+{kibana-ref}/canvas-getting-started.html[Canvas workpad].
+
+* If you’re interested in business analytics, install the set of *sample eCommerce orders*, 
+which includes visualizations for product-related information, such as cost, revenue, and price. 
+* If you’re looking to analyze website traffic, check out the *sample web logs* data set. 
+* If you want to check out some data with geo information, you can install the 
+*sample flight data* to view and interact with data that contains flight routes for four airlines. 
+
+*Note:* The timestamps included in the sample data sets are relative to when they are installed. If you uninstall and reinstall a data set, the timestamps will change to reflect the most recent installation.
```

Comment:
How about moving  the note after the sentence that starts with Once. 

The timestamps included in -> The timestamps in